traffic_ops/traffic_ops_golang/dbhelpers/db_helpers.go

```diff
@@ -42,6 +42,75 @@ const BaseOrderBy = "\nORDER BY"
 const BaseLimit = "\nLIMIT"
 const BaseOffset = "\nOFFSET"
 
+const UserIDHasAccessToDeliveryServiceXMLIDQuery = `
```

Comment:
You know it might not. I think that's a remnant from the old way I wanted to do it, but I couldn't figure out the query for it (that one doesn't work). I'll remove it.